### PR TITLE
OEL-2598: Add optional categories to news items.

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: tests/src/ExistingSite/NewsTest.php [hunk 1]
```diff
@@ -30,6 +35,45 @@ protected function setUp(): void {
...
 
+  /**
+   * Test News type CRUD.
+   */
+  public function testNewsType(): void {
+    $assert_session = $this->assertSession();
+    $page = $this->getSession()->getPage();
+
+    // Assert user without permission can't create news types.
+    $this->assertPathsRequireRole([
```
Comment (brummbar) [inline tests/src/ExistingSite/NewsTest.php]: Please check how this has been done in https://github.com/openeuropa/oe_showcase/pull/202.

File: tests/src/ExistingSite/NewsTest.php [hunk 2]
```diff
@@ -60,6 +105,13 @@ public function testCreateNews() {
     ]);
     $media_image->save();
 
+    for ($i = 1; $i < 4; $i++) {
+      Term::create([
+        'vid' => 'news_type',
+        'name' => "News type $i",
+      ])->save();
+    }
```
Comment (brummbar) [inline tests/src/ExistingSite/NewsTest.php]: Please follow https://github.com/openeuropa/oe_showcase/pull/202/files#diff-3b4fc7927c226f71dbc0f511f9c6c951714e1b0f1109cc563db388445b65191aR33 .

**Review by brummbar — CHANGES_REQUESTED** [review]
Missing list pages integration.
Fields should be put inside the oe_showcase_fields module.

File: tests/src/ExistingSite/NewsTest.php (hunk 1 above)
Comment (AaronGilMartinez) [reply tests/src/ExistingSite/NewsTest.php]: Moved

File: tests/src/ExistingSite/NewsTest.php (hunk 2 above)
Comment (AaronGilMartinez) [reply tests/src/ExistingSite/NewsTest.php]: Done

**Comment by AaronGilMartinez** [comment]
> Missing list pages integration. Fields should be put inside the oe_showcase_fields module.

Added filter for news list page and moved field to module

File: modules/oe_showcase_fields/oe_showcase_fields.post_update.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * @file
+ * OpenEuropa Showcase Fields post updates.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\oe_bootstrap_theme\ConfigImporter;
+
+/**
+ * Add News type to News.
```
Comment (brummbar) [inline modules/oe_showcase_fields/oe_showcase_fields.post_update.php]: :D

**Review by brummbar — CHANGES_REQUESTED** [review]
Just this small remark :D

**Review by brummbar — APPROVED** [review]

**Pull request merged**